src/components/drivers/pdf-viewer.jsx

```diff
@@ -9,15 +9,23 @@ PDFJS.disableWorker = true;
 const INCREASE_PERCENTAGE = 0.2;
 const DEFAULT_SCALE = 1.1;
 
-class PDFPage extends React.Component {
-
+export class PDFPage extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
     this.onChange = this.onChange.bind(this);
   }
 
+  componentDidMount() {
+    if (this.props.disableVisibilityCheck) this.fetchAndRenderPage();
```

Comment:
why is this necessary?

Reply:
The pdf was getting rendered before only in `componentDidUpdate` based of that visibility check, so now we can/have to render the pdf into the DOM right away.